dbms/src/TiDB/Schema/SchemaBuilder.h

```diff
@@ -88,6 +88,10 @@ struct SchemaBuilder
 
     void applySetTiFlashReplica(TiDB::DBInfoPtr db_info, TableID table_id);
     void applySetTiFlashReplica(TiDB::DBInfoPtr db_info, TiDB::TableInfoPtr table_info, ManageableStoragePtr storage);
+
+    void applySetTiFlashMode(TiDB::DBInfoPtr db_info, TableID table_id);
+    void applySetTiFlashModeOnLogicalTable(TiDB::DBInfoPtr db_info, TiDB::TableInfoPtr table_info, ManageableStoragePtr storage);
+    void applySetTiFlashModeOnPhysicalTable(TiDB::DBInfoPtr db_info, TiDB::TableInfoPtr table_info, ManageableStoragePtr storage);
```

Comment:
```suggestion
    void applySetTiFlashMode(const TiDB::DBInfoPtr & db_info, TableID table_id);
    void applySetTiFlashModeOnLogicalTable(const TiDB::DBInfoPtr & db_info, const TiDB::TableInfoPtr & table_info, ManageableStoragePtr storage);
    void applySetTiFlashModeOnPhysicalTable(const TiDB::DBInfoPtr & db_info, const TiDB::TableInfoPtr & table_info, ManageableStoragePtr storage);
```

Make db_info and table_info to be `const ... &` to avoid copying